applications/mne_scan/plugins/plugins.pro

```diff
@@ -43,6 +43,7 @@ TEMPLATE = subdirs
 #Sensors
 SUBDIRS += \
     fiffsimulator \
+    ftbuffer \ #needs fieldtrip buffer source code
```

Comment:
I think the comment `#needs fieldtrip buffer source code` can be removed, right?